tests/sles4sap/sap_suse_cluster_connector.pm

```diff
@@ -55,31 +55,33 @@ sub run {
 
     # Check HA config and list resources
     my @commands = get_var('NW') ? qw(hcc lsr) : qw(hcc);
-    exec_conn_cmd(binary => $binary, cmd => "$_ --sid $instance_sid --ino $instance_id", log_file => $log_file) foreach (@commands);
+    exec_conn_cmd(binary => $binary, cmd => "$_ --sid ${instance_sid} --ino ${instance_id}", log_file => $log_file) foreach (@commands);
 
     # Test Maintenance Mode
     foreach my $mod (1, 0) {
-        exec_conn_cmd(binary => $binary, cmd => "smm --sid $instance_sid --ino $instance_id --mod $mod", log_file => $log_file);
+        exec_conn_cmd(binary => $binary, cmd => "smm --sid ${instance_sid} --ino ${instance_id} --mod ${mod}", log_file => $log_file);
         # Wait to let enough time for the HA stack to change Maintenance Mode
         sleep 10;
     }
 
     # List nodes
-    my @resources = get_var('NW') ? ('ip', 'fs', 'sap') : ('ip', 'SAPHanaTopology', 'SAPHana');
+    my @hana_resources = get_var('USE_SAP_HANA_SR_ANGI') ? ('ip', 'SAPHanaTpg', 'SAPHanaCtl') : ('ip', 'SAPHanaTopology', 'SAPHana');
+    my @resources = get_var('NW') ? ('ip', 'fs', 'sap') : @hana_resources;
     foreach my $rsc_type (@resources) {
         my $rsc = "rsc_${rsc_type}_${instance_sid}_${instance_type}${instance_id}";
         wait_for_idle_cluster;
-        exec_conn_cmd(binary => $binary, cmd => "lsn --res $rsc", log_file => $log_file);
+        exec_conn_cmd(binary => $binary, cmd => "lsn --res ${rsc}", log_file => $log_file);
```

Comment:
Change not needed.

Reply:
Curly braced variable is safely enclosed variable.